doc/user/layouts/partials/create-source/connector/kafka/syntax.html

```diff
@@ -3,3 +3,5 @@
 **INCLUDE PARTITION** | Include a `partition` column containing the Kafka message partition. The column can be renamed with the optional **AS** *name* clause.
 **INCLUDE OFFSET** | Include an `offset` column containing the Kafka message offset. The column can be renamed with the optional **AS** *name* clause.
 **INCLUDE TIMESTAMP** | Include a `timestamp` column containing the Kafka message timestamp. The column can be renamed with the optional **AS** *name* clause. <br><br>Note that the timestamp of a Kafka message depends on how the topic and its producers are configured. See the [Confluent documentation](https://docs.confluent.io/3.0.0/streams/concepts.html?#time) for details.
+**INCLUDE HEADERS** | Include a `header` column containing the Kafka message headers as a list of records of type `(key text, value bytea)`. The column can be renamed with the optional **AS** *name* clause.
+**INCLUDE HEADER** _key_ **AS** _name_ | Include a _name_ column containing the Kafka message header _key_ parsed as a UTF-8 string. The header values can alsp be exposed as `bytea` with the optional **BYTES** flag.
```

Comment:
```suggestion
**INCLUDE HEADER** _key_ **AS** _name_ [**BYTES**] | Include a _name_ column containing the Kafka message header _key_ parsed as a UTF-8 string. The header value can alternatively be exposed with type `bytea` with the optional **BYTES** flag.
```

Reply:
Is there a specific reason why we made aliasing the header key for this option mandatory? Couldn't find the context in the original PR.